doc/language-onboarding.md

```diff
@@ -100,7 +100,7 @@ The container is expected to produce up to two artifacts:
 {
   "libraries": [
     {
-      "id": "google-cloud-secretmanager",
+      "id": "secretmanager",
```

Comment:
Can you move this to after pubsub? We started keeping state.yaml in alphabetical order a while ago, and I suspect the request will conform to that.

Reply:
Done.